signals.go

```diff
@@ -55,15 +58,28 @@ func backtrace() {
 }
 
 func fatalSignal(sig syscall.Signal) bool {
-	return sigFatal[sig]
+	_, exists := handledSignalsMap[sig]
+	if !exists {
+		return false
+	}
+
+	return handledSignalsMap[sig]
+}
+
+func nonFatalSignal(sig syscall.Signal) bool {
+	_, exists := handledSignalsMap[sig]
+	if !exists {
+		return false
+	}
+
+	return !handledSignalsMap[sig]
```

Comment:
I think you can improve the performance just a little , see https://github.com/kata-containers/runtime/pull/244  :smile: